tests/sample_test.py

```diff
@@ -0,0 +1,4 @@
+import pytest
+
+def test_answer():
+    assert 5 == 5
```

Comment:
Placeholder test so that pytest would run for Travis. This will be replaced with actual tests of code ASAP.